### ISSUE TITLE
SA: option for custom outputs

### ISSUE THREAD

**Issue by PavelBal** [esqLABS/esqlabsR#622]
SA should support other outputs than PK-parameters only.

How? Proposal - the user provides a function with well-defined signature and returns a value. E.g.:

```
customSAOuput <- function(simulationResults, outputPath) {
  tmp <- simulationResults$data$outputPath
  # do some magic
  result <- magic(tmp)
  return(result)
}
```

Checks to perform:

- function signature is `SimulationResults, string`

**Comment by Laura-Villain** [comment]
Example: average of tmp between times X and Y (ex: mean tmp between time=10 min and time =60 min)

**Comment by PavelBal** [comment]
When implemented, update the plotting functions.

- Update documentation
- Add `customOutputs` argument

**Comment by rengelke** [comment]
Resolved in #685
